tune/protox/env/util/pg_conn.py

```diff
@@ -208,12 +220,59 @@ def start_with_changes(
                 "Waiting for postgres to bootup but it is not..."
             )
 
-        # Move the temporary over since we know the temporary can load.
+        # Set up Boot if we're told to do so
```

Comment:
Good point, that's something I didn't realize. I think there is an argument for why Proto-X should know about Boot though, because when you use Proto-X to tune, you need to indicate whether Boot was used during tuning or not to know whether the numbers are real runtimes or estimated. We can revisit this later though.